### ISSUE TITLE
FeatureFileDumper main method could handle exceptions better

### ISSUE THREAD

**Issue by davidmoten** [cmusphinx/sphinx4#18]
The main method in ```FeatureFileDumper``` catches exceptions and writes to ```System.err``` but does not exit the process with a non-zero exit code nor throw a ```RuntimeException```. One of these should occur.

```java
        try {
            ...
        } catch (IOException ioe) {
            System.err.println("I/O Error " + ioe);
        } catch (PropertyException p) {
            System.err.println("Bad configuration " + p);
        }
```

**Comment by nshmyrev** [comment]
Thank you for your report. In processing large databases it's ok to have issues. Stopping the work is not a good idea the, it's better to warn and proceed. The filedumper needs rewokr though.

**Comment by davidmoten** [comment]
I agree that sometimes the caller would want problems to be ignored but it
should be up to the caller to do the ignoring (or not ignore). Putting
output to stderr only is not a great way of signaling a problem in my
opinion.

My reading indicates that the consensus for main method usage is to call
System.exit as the main method is expected to be called from scripts rather
than Java code. Scripts can ignore errors if they want to.
